google/cloud/storage/emulator/README.md

```diff
@@ -84,3 +84,61 @@ For N==1 and N==2 behave like `return-305-after-256K`, for `N>=3` ignore the
 failure instruction and return successfully. This is used to test failures during
 retry, the client cooperates by sending the retry counter in the failure
 instructions.
+
+
+## Retry Test API
+
+The GCS Emulator at the time of writing (5/19/21), only provided support
+for reproducing transient errors by passing instructions through a header
+which is not easily implementable across the 8 language libraries that Cloud
+Storage currently supports. Retry Test API introduces a method to supply failure
+configuration and supplies GCS libraries with a unique identifier which is
+used to identify how to fail a specified GCS API operation. In addition,
```

Comment:
This second motivation seems important enough by itself, and I assume the logs are more readable too.